Controller/ContentTypeController.php

```diff
@@ -216,6 +233,25 @@ public function viewContentTypeAction($contentTypeId, $languageCode = null)
         $contentCount = $this->searchService->findContent($countQuery, [], false)->totalCount;
         $deleteForm = $this->createForm(new ContentTypeDeleteType(), ['contentTypeId' => $contentTypeId]);
 
+        if (isset($languageCode) && !isset($contentType->names[$languageCode])) {
+            $this->notify(
```

Comment:
As an alternative I tried doing a `redirectToRouteAfterFormPost()` after the notification. This generates a HTTP 205 with the correct new pjax-location (with changed language), looks good. But the page never loads, it just spins the loader icon. It's not an infinite loop, I've checked that.